tests/unit/core/core.js

```diff
@@ -138,4 +138,89 @@ test( "uniqueId / removeUniqueId", function() {
 	equal( el.attr( "id" ), null, "unique id has been removed from element" );
 });
 
+// Support: Core 1.9 Only, IE8 Only
+// The use of $.trim() in the two tests below this is to account for IE8 missing string.trim()
```

Comment:
It's used 5 times in 2 tests all on different lines so i was not sure the best place to put it. I did want to duplicate 5 times and putting it in the first test at first use seemed wrong since it would have no apparent connection to second test. Thoughts?